loki/transformations/inline.py

```diff
@@ -606,6 +606,18 @@ def inline_marked_subroutines(routine, allowed_aliases=None, adjust_imports=True
                 # Remove import if no further symbols used, otherwise clone with new symbols
                 import_map[impt] = impt.clone(symbols=new_symbols) if new_symbols else None
 
+        # Remove explicit interfaces of inlined routines
+        for intf in FindNodes(Interface).visit(routine.ir):
```

Comment:
Do we really need to go over `routine.ir` here? Or couldn't we simply use 
```suggestion
        for intf in routine.interfaces:
```

Reply:
As discussed offline, this can be dropped now because `InlineTransformation.creates_items = True` if `inline_marked = True`.